i18n/messages.en.js

```diff
@@ -319,8 +320,9 @@ module.exports = {
     prerequisites: {
       body: `<p><b>Recommended prerequisites</b> describe what the teacher expects the student to have mastered when the course begins.</p>
       <p><b>This information helps the student</b> understand which knowledge is crucial to be updated on when the course starts. Clearly defined information makes it easier for the student to review the correct theories and models before the course begins.</p>
-      <p><b>Content is retrieved from Kopps</b> and can only be edited in <a href="https://app.kth.se/kopps/admin/courses" target="_blank">Kopps</a> by <a href="https://app.kth.se/kopps/admin/userlist" target="_blank">authorized personnel</a> (both links open in new tabs). You can choose not to include the heading in your course memo.</p>`,
+      <p><b>The content is retrieved from the Page Before course selection</b> and can only be edited via the About course administration tool. If you want to change the content, you need to save your course memo as a draft and go to the function <a href="empty" target="_blank">Page Before course selection</a > to edit the text. You can choose not to include the title in your course memo.</p>`,
```

Comment:
Tiny comment. Fix if you agree, leave if not. Think this would look better:

`from the page Before course selection`

and

`to the function page <a href="empty" target="_blank">Before course selection</a >`

with lowercase "p" and move "page" to outside the link.